packages/config/config/devices/0x027a/zen16.json

```
@@ -74,6 +74,13 @@
 			"$import": "templates/zooz_template.json#zen16_switch_type_v2",
 			"label": "Switch 1: Type"
 		},
+		{
+			"#": "25",
+			"$if": "firmwareVersion >= 1.2",
+			"$import": "templates/zooz_template.json#zen16_switch_invert_v2",
+			"label": "Switch 1: Invert Value",
+			"description": "Invert the reported values for your selected switch type in Sw1 terminals (select the same value as you chose for parameter 2 to invert the values for open and closed circuit)."
```

We try to keep descriptions as short as possible and avoid repeating the label in other words. Only use this if necessary to add additional information, e.g.
```suggestion
			"description": "Select the same value as in parameter 2 to invert the values for open and closed circuit."
```

Same for params 26/27